posthog/models/async_deletion/delete.py

```diff
@@ -22,8 +22,37 @@
 
 
 def run_event_table_deletions():
-    queued_deletions = list(AsyncDeletion.objects.filter(delete_verified_at__isnull=True))
-    process_found_event_table_deletions(queued_deletions)
+    queued_event_deletions = AsyncDeletion.objects.filter(
+        delete_verified_at__isnull=True, deletion_type__in=[DeletionType.Team, DeletionType.Person, DeletionType.Group]
+    )
+    process_found_event_table_deletions(list(queued_event_deletions))
+
+    queued_cohortpeople_deletions = AsyncDeletion.objects.filter(
+        delete_verified_at__isnull=True, deletion_type__in=[DeletionType.Cohort_full, DeletionType.Cohort_stale]
```

Comment:
Doesn't look like we've added to _verify_by_group which wouldn't verify these deletions?